packages/next/build/webpack/plugins/profiling-plugin.ts

```diff
@@ -1,13 +1,28 @@
 import { tracer } from '../../tracer'
+import { webpack, isWebpack5 } from 'next/dist/compiled/webpack/webpack'
 
 const pluginName = 'ProfilingPlugin'
 
 export const spans = new WeakMap()
 
+function getNormalModuleLoaderHook(compilation: any) {
+  if (isWebpack5) {
+    // @ts-ignore TODO: Remove ignore when webpack 5 is stable
+    return webpack.NormalModule.getCompilationHooks(compilation).loader
+  }
+
+  return compilation.hooks.normalModuleLoader
+}
+
 export class ProfilingPlugin {
   apply(compiler: any) {
     // Only enabled when instrumentation is loaded
-    if (!tracer.getCurrentSpan()) {
+    const currentSpan = tracer.getCurrentSpan()
+    console.log({
+      currentSpan,
+      isRecording: currentSpan?.isRecording(),
+    })
```

Comment:
@timneutkens 
Is this `console.log` kept intentionally?

Reply:
It wasn't, totally missed this!